client/my-sites/earn/home.tsx

```diff
@@ -187,6 +188,74 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 		};
 	};
 
+	/**
+	 * Return the content to display in the Premium Content Block card based on the current plan.
+	 *
+	 * @returns {object} Object with props to render a PromoCard.
+	 */
+	const getPremiumContentCard = () => {
+		const cta = isFreePlan
+			? {
+					text: translate( 'Unlock this feature' ),
+					action: () => {
+						trackUpgrade( 'any-paid-plan', 'premium-content' );
+						page( `/plans/${ selectedSiteSlug }` );
+					},
+			  }
+			: {
+					text: translate( 'Add premium content subscriptions' ),
+					action: () => {
+						trackCtaButton( 'premium-content' );
+						page( '/earn/payments-plans/${ selectedSiteSlug }' );
+					},
+			  };
+		const title = hasConnectedAccount
+			? translate( 'Manage Your Premium Content' )
```

Comment:
We recently made a decision to write headings in sentence case, so this should be `Manage your premium content`. It will also keep it consistent with the rest of the headings in this section.

Reply:
Got it, I updated both the recurrent payments and premium content block card titles to use sentence case.

Reply:
Got it. I update both the premium content and recurring payments card title to sentence case.